apps/meteor/client/sidebar/header/hooks/useVoipItems.tsx

```diff
@@ -0,0 +1,67 @@
+import { Box } from '@rocket.chat/fuselage';
+import type { GenericMenuItemProps } from '@rocket.chat/ui-client';
+import { useToastMessageDispatch } from '@rocket.chat/ui-contexts';
+import { useVoipAPI, useVoipState } from '@rocket.chat/ui-voip';
+import { useMutation } from '@tanstack/react-query';
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+
+const useVoipItems = (): GenericMenuItemProps[] => {
```

Comment:
Isn't this the exact same code in NavBarV2 folder?

Reply:
It is. I saw that the other hooks were duplicated as well, so I assumed we are not creating dependencies between the old and new nav bar.

Reply:
I too saw this pattern when working in another issue. I think it's ok then if we're commited to this.